debian/publish.sh

```diff
@@ -5,7 +5,7 @@
 # DEBIAN_REPO_PASSWORD='password'
 
 DEBIAN_REPO_BASE_URL="https://openmrs.jfrog.io/artifactory/deb-pih/pool"
-DEBIAN_REPO_DISTRIBUTION=unstable
+DEBIAN_REPO_DISTRIBUTION=$1
```

Comment:
Might be nice to have a check to validate that either "unstable" or "stable" is passed in, and have usage documented around this.

Reply:
Good call, updated